scripts/generate_gift_cards_for_campaign.rb

```diff
@@ -0,0 +1,20 @@
+def max_campaign(payment_intent_id, campaign_id)
+  Item.transaction do
+    campaign = Campaign.find(campaign_id)
+    payment_intent = PaymentIntent.find(payment_intent_id)
+    payment_intent.update!(recipient: campaign.distributor.contact)
+
+    (1..(campaign.target_amount / campaign.price_per_meal)).each do |_|
```

Comment:
do we have to take into account the amount already raised? (I'm not sure what the use case for this script is -- from the method name my understanding is that it's purpose is to max out an existing campaign?)